code/modules/jobs/job_types/engineering.dm

```diff
@@ -77,7 +77,7 @@ Station Engineer
 	access = list(access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels,
```

Comment:
In my opinion cook should not be using those bodies to make food, there are a lot of alternatives. Also scientist doesn't need tech access because that should mainly be for engineers who need to fix a bombed place etc.